### PR TITLE
as in Plone, hide controlpanel for users that are no 'Manager' or 'Si…

### PR DESCRIPTION
**Pull request by giuliaghisini**
…te Administrator'

### CODE REVIEW

File: src/components/manage/Toolbar/PersonalTools.jsx [hunk 1]
```diff
@@ -143,12 +145,17 @@ class PersonalTools extends Component {
                 <Icon name={rightArrowSVG} size="24px" />
               </button>
             </li>
-            <li>
-              <Link to="/controlpanel">
-                <FormattedMessage id="Site Setup" defaultMessage="Site Setup" />
-                <Icon name={rightArrowSVG} size="24px" />
-              </Link>
-            </li>
+            {isControlpanelEnabled(this.props.user) && (
```
Comment (tiberiuichim) [inline src/components/manage/Toolbar/PersonalTools.jsx:148]: Maybe `isAdminUser` ?

Comment (giuliaghisini) [reply src/components/manage/Toolbar/PersonalTools.jsx]: Yes! It's better. I've updated the code

**Review by nzambello — APPROVED** [review]

**Comment by cekk** [comment]
i think that we should start thinking about how to check user permissions and not roles with restapi, because we are missing a lot of Plone's power like this.

File: src/helpers/User/User.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * Is admin user method.
+ * @method isAdminUser
+ * @returns {undefined}
+ */
+export function isAdminUser(user) {
+  return user?.roles
+    ? user.roles.includes('Site Administrator') ||
+        user.roles.includes('Manager')
+    : false;
+}
```
Comment (pnicolli) [inline src/helpers/User/User.js:16]:
```suggestion
import { intersection } from 'lodash/intersection';

/**
 * Checks if the user has one or more given roles.
 * @method userHasRoles
 * @returns {boolean}
 */
export function userHasRoles(user, roles = []) {
  return intersection(user?.roles ?? [], roles).length > 0;
}
```

How about a slightly different and more generic approach, like this for example? I can't decide whether I would prefer having a direct ``isAdminUser`` method or a more generic one, or maybe both.

**Pull request merged**